pkg/cli/server/config_reloader_test.go

```diff
@@ -508,11 +510,11 @@ func TestConfigReloader(t *testing.T) {
 		So(string(data), ShouldContainSubstring, "\"UpdateInterval\":18000000000000")
 		So(string(data), ShouldContainSubstring, "\"Scrub\":null")
 		So(string(data), ShouldContainSubstring, "\"DBRepository\":\"another/unreachable/trivy/url2\"")
-		// matching log message when it errors out, test that indeed the download will try the second url
-		found, err = test.ReadLogFileAndSearchString(logFile.Name(),
```

Comment:
This function reads the log file in a loop until the string is found (with 1 minute timeout).
In the new code there is no guarantee the background task attempting the download is done within the initial 5 seconds.

Reply:
Reverting to the old test with new string.

Reply:
Apparently, "data" is used mainly to check the first log message shown when the config is loaded. 5 is supposed to be safe to assume that message shows. The other message is supposed to verify the new configuration is actually used by the trivy logic. That is a background task run by the scheduler, so a function reading the logfile in a loop was used to make sure the test waits enough.